_schema.graphql

```diff
@@ -4497,6 +4499,7 @@ type HomePageModuleContextFair {
   # A formatted description of the start to end dates
   exhibition_period: String
   has_full_feature: Boolean
+  has_incorrect_snake_case: Boolean
```

Comment:
<!--
  1 failure:  Found addition us...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: _schema.graphql;
  Line: 4502;
-->  
- :no_entry_sign: Found addition using snake_case instead of camelCase: has_incorrect_snake_case